### PR TITLE
Added release notes for release-6.7

### PR DESCRIPTION
**Pull request by harigowtham**
Signed-off-by: hari gowtham <hgowtham@redhat.com>

### CODE REVIEW

File: docs/release-notes/6.7.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Release notes for Gluster 6.7
+
+This is a bugfix release. The release notes for [6.0](6.0.md), [6.1](6.1.md),
+ [6.2](6.2.md), [6.3](6.3.md), [6.4](6.4.md), [6.5](6.5.md) and [6.6](6.6.md)
+contains a listing of all the new features that were added
```
Comment (sankarshanmukhopadhyay) [inline docs/release-notes/6.7.md]: 'contain' a listing of all the new features that have been added along with bugs fixed in the GlusterFS 6 stable release.

File: docs/release-notes/6.7.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+and bugs fixed in the GlusterFS 6 stable release.
+
+**NOTE:** Next minor release tentative date: Week of 30th February, 2020
```
Comment (sankarshanmukhopadhyay) [inline docs/release-notes/6.7.md]: Tentative date for next minor release

Comment (harigowtham) [reply docs/release-notes/6.7.md]: done

File: docs/release-notes/6.7.md (hunk 1 above)
Comment (harigowtham) [reply docs/release-notes/6.7.md]: done

**Comment by harigowtham** [comment]
Had the date mentioned as Feb 30 and have fixed it.

**Pull request merged**